config/packages/doctrine.yaml

```diff
@@ -30,4 +30,4 @@ doctrine:
       numeric_functions:
         time_diff: App\DQL\TimeDiff
         time_add: App\DQL\TimeAdd
-        time_sub: App\DQL\TimeSub
\ No newline at end of file
+        time_sub: App\DQL\TimeSub
```

Comment:
I noticed throughout this PR that sometimes such changes happened. Was there just a linebreak inserted? 😄